src/main/frontend/state.cljs

```diff
@@ -1016,15 +1017,7 @@
 
 (defn get-date-formatter
   []
-  (or
-    (when-let [repo (get-current-repo)]
-      (or
-        (get-in @state [:config repo :journal/page-title-format])
-        ;; for compatibility
-        (get-in @state [:config repo :date-formatter])))
-    ;; TODO:
-    (get-in @state [:me :settings :date-formatter])
```

Comment:
Migrating this would take effort like the stateful formats. Removed it since it seemed to never be used